libs/utils/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ set(DIST_HDRS
         ${PUBLIC_HDR_DIR}/${TARGET}/SpinLock.h
         ${PUBLIC_HDR_DIR}/${TARGET}/StructureOfArrays.h
         ${PUBLIC_HDR_DIR}/${TARGET}/unwindows.h
+        ${PUBLIC_HDR_DIR}/${TARGET}/memalign.h
```

Comment:
Please keep the list of files alphabetically ordered.

Reply:
Done